### ISSUE TITLE
GUI is not redrawn after s2ram

### ISSUE THREAD

**Issue by JohannesLorenz** [zynaddsubfx/zyn-fusion-issues#361]
When I wake up my PC from suspend 2 ram, the GUI is not redrawn, and thus it's mostly white with some weird drawings. You have to click around a a bit to hit the right widgets. I.e. clicking the envelope UI lets it show up the envelope, and clicking on part redraws the whole UI.

**Comment by fundamental** [comment]
Right now the GUI assumes that the framebuffers used to draw the individual layers remain valid unless there has been movement in those rectangular regions. Off the top of my head I don't know how the application should identify that those buffers have been damaged by a sleep and wake combination.

**Comment by JohannesLorenz** [comment]
The GUI behind zyn-fusion is OpenGL, right? So this problem would exist with every OpenGL app?

**Comment by fundamental** [comment]
Yep. I know other applications have historically had the exact same issue.
I.e. gl buffer content is destroyed on sleep. I do not know if other
impacted applications have found a workaround
